much-slug.gemspec

```
@@ -15,15 +15,16 @@ Gem::Specification.new do |gem|
   gem.homepage    = "https://github.com/redding/much-slug"
   gem.license     = "MIT"
 
-  gem.files         = `git ls-files`.split($/)
+  gem.files         = `git ls-files`.split($INPUT_RECORD_SEPARATOR)
   gem.executables   = gem.files.grep(%r{^bin/}).map{ |f| File.basename(f) }
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.require_paths = ["lib"]
 
   gem.required_ruby_version = "~> 2.5"
 
-  gem.add_dependency("much-mixin", ["~> 0.2.3"])
+  gem.add_development_dependency("ardb",             ["~> 0.29.1"])
+  gem.add_development_dependency("assert",           ["~> 2.19.2"])
+  gem.add_development_dependency("much-style-guide", ["~> 0.4.0"])
```

```suggestion
  gem.add_development_dependency("much-style-guide", ["~> 0.5.0"])
```

---

```suggestion
  gem.add_development_dependency("much-style-guide", ["~> 0.6.0"])
```
(I think I'm done for now - sorry for the quick iterations)